### PR TITLE
65 text styling

### PR DESCRIPTION
**Pull request by loganhochwald**
Add some stylings, would like to discuss if anything should be altered. Nothing too major.

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
+import { Card, CardBody, CardHeader, Divider } from "@nextui-org/react";
 
 import React from "react";
 
 export default function Home() {
   return (
-    <div className="flex justify-center items-start h-screen">
-      <Card isBlurred className="mt-24 max-w-md" shadow="lg">
+    <div className="flex justify-center items-start h-screen p-4">
+      <Card isBlurred className="mt-24 max-w-md p-4 border border-stone-300" shadow="lg">
```
Comment (colinwilliams91) [inline app/page.tsx:9]: yay, i like the border

**Comment by colinwilliams91** [comment]
Thanks @loganhochwald!

I've been trying to enforce [semantic commit messaging](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716) for easier cherry picking and reverse commit lookup as needed. Your commits are good but it will just be easier if we have a universal convention/categories to search by for the future.

[why use conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#why-use-conventional-commits)

conventional commits is a widely used revision upon [angular commit message format](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#-commit-message-format)

**Review by colinwilliams91 — APPROVED** [review]
nice! paddings, margins, justifies, all nice things that I overlooked 👍 
- [x] ready for merge

File: app/page.tsx [hunk 2]
```diff
@@ -5,8 +5,8 @@ import React from "react";
 
 export default function Home() {
   return (
-    <div className="flex justify-center items-start h-screen p-4">
```
Comment (codingMustache) [inline app/page.tsx]: when you add h-screen here you make this div as big as the screen + the nav bar sits on top so it was doing some weird scrolling stuff

File: app/page.tsx [hunk 3]
```diff
@@ -5,8 +5,8 @@ import React from "react";
...
-      <Card isBlurred className="mt-24 max-w-md p-4 border border-stone-300" shadow="lg">
```
Comment (codingMustache) [inline app/page.tsx]: When I removed h-screen I changed this to be 16 to compensate for the removed div size

**Pull request merged**